docs/howto/how-to-contribute.md

```diff
@@ -24,6 +24,7 @@ You can also look at our [issues](https://github.com/Samsung/CredSweeper/issues)
 
 For small changes, you can use GitHub's web interface. Simply click the **Edit the file in your fork of this project** on the file you'd like to change.
 GitHub creates the new branch for you when you submit the changes.
+VCS(git) requirement: the branch MUST be forked after latest release.
```

Comment:
Can you please comment on why is that?
Can't I just fetch main branch after the release?